doc/crm.8.adoc

```diff
@@ -932,6 +932,42 @@ Usage:
 verify [scores]
 ...............
 
+[[cmdhelp_insight,Cluster insight]]
+=== `insight` - Cluster insight
+
+Cluster insight help details
+
+[[cmdhelp_insight_preflight_check,Cluster preflight check tool set]]
+==== `preflight_check` - Cluster preflight check tool set
+
+To trigger cluster failures and verify some key configuration 
+right before it goes to production. Cluster failure simulation 
+by different people can go wild and the results can be different, 
+even wrongly interpreted. This tool standardizes the steps as the simple CLI. 
+It is carefully designed and must not change any configuration to harm the production.
+
+usage: preflight_check [-c]
+                       [--kill-sbd | --kill-corosync | --kill-pacemakerd | --fence-node NODE | --split-brain-iptables]
+                       [-l] [-y] [-h]
+
+Cluster preflight check tool set
+
+optional arguments:
+  -c, --check-conf        Valid the configurations
+  --kill-sbd              Kill sbd daemon
+  --kill-corosync         Kill corosync daemon
+  --kill-pacemakerd       Kill pacemakerd daemon
+  --fence-node NODE       Fence specific node
+  --split-brain-iptables  Make split brain by blocking corosync ports
+  -l, --kill-loop         Kill process in loop
+
+other options:
+  -y, --yes               Answer "yes" if asked to run the test
+  -h, --help              Show this help message and exit
+
+Log: /var/log/crmsh/preflight_check.log
+Json results: /var/lib/crmsh/preflight_check/preflight_check.json
+For each --kill-* testcase, report directory: /var/lib/crmsh/preflight_check
```

Comment:
> Seems like the report is not provided to --fence-node, --kill-loop. The impression is not consistent to others. Would be better to share a brief reason?

@zzhou1 
I thought `--fence-node` only have one fix result, no need to add more explanations for the possible different results like --kill-*; And the same with `--kill-loop`, kill one process in looping will leading fencing, still has only one result

If we really have requirements to add these report for these options, I suggest to add this into backlog/TODO list

> 
> Seems like `preflight_check.json` is overwritten everytime. Do you consider "preflight_check-20210125-1611559728.json"?
> 

Yes, so far this behavior was by design. My thinking was if upper application calling one action of preflight_check, it can get the json result immediately, any order json results might not useful.
We can discuss and add more thinking to backlog if need to change or has strong requirements.

> And mayabe rephrase the help message:
> 
> The results with JSON and TXT format are under: /var/lib/crmsh/preflight_check

Reply:
No hard requirement. Instead, they are my first impression, and could be considered to incooperate these feedback to shape the tool more intuitive to users.

> > Seems like the report is not provided to --fence-node, --kill-loop. The impression is not consistent to others. Would be better to share a brief reason?
> 
> @zzhou1
> I thought `--fence-node` only have one fix result, no need to add more explanations for the possible different results like --kill-*; And the same with `--kill-loop`, kill one process in looping will leading fencing, still has only one result
> 
> If we really have requirements to add these report for these options, I suggest to add this into backlog/TODO list
> 

My impression is the result is inconsistent, which more often means confusing, and trigger questions.